### ISSUE TITLE
[Feature Request] Expand/Collapse All in TO&E Screen

### ISSUE THREAD

**Issue by Kendricke** [MegaMek/mekhq#302]
As a GM/Unit Commander, I would like a right-click option to expand or collapse all forces and subforces under a particular force icon in the TO&E screen.  This is especially useful the larger or more complex my overall unit becomes.  

(Additionally, it would be great to have MekHQ remember the state I left the forces in each time I save the campaign).

**Comment by purpledragon2** [comment]
+1 on both points.   I bet a separate Request Feature will be requested for the second point.

**Comment by neoancient** [comment]
Correct. These are two different features, and should be requested in separate issue tickets.

**Comment by Kendricke** [comment]
Feature Request #303 opened for the second request.
